src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -16,19 +38,26 @@ export const TodoFilter = () => (
         type="text"
         className="input"
         placeholder="Search..."
+        value={query}
+        onChange={(event) => onQueryChange(event.target.value)}
```

Comment:
Move to separate function  (transfer the function to a variable, and here only write the name of the function)